src/main/kotlin/game/Car.kt

```diff
@@ -1,16 +1,39 @@
 package game
 
-class Car(var position: Int = 0) {
+class Car private constructor(private var position: Int = 0, val name: String) {
 
     fun move(moveCondition: Int) {
         when {
             canMove(moveCondition) -> advance()
         }
     }
 
+    fun isSamePosition(position: Int) = this.position == position
+
     private fun advance() {
         position += 1
     }
 
+    fun getPosition(): Int {
+        return position
+    }
```

Comment:
```suggestion
    val position: Int
    get() = _position
```
코틀린에서는 getter함수를 선언하기보단, 다양한 방법이 있어요! 
backing-fields 를 활용해보아요! 
https://kotlinlang.org/docs/properties.html#backing-fields